apps/worker/src/app/workflow/services/active-jobs-metric.service.ts

```diff
@@ -66,7 +65,7 @@ export class ActiveJobsMetricService {
               repeatJobKey: METRIC_JOB_ID,
               repeat: {
                 immediately: true,
-                pattern: '* * * * * *',
+                pattern: '*/30 * * * * *', // Every 30 seconds
```

Comment:
We have a `CronExpressionEnum` defined [here](https://github.com/novuhq/novu/blob/next/packages/shared/src/types/cron/cron.types.ts#L9) that we might like to use.

Reply:
#til. Updated :)